cylc/flow/subprocpool.py

```diff
@@ -554,8 +554,15 @@ def rsync_255_fail(ctx, platform=None) -> bool:
         the failures may be caused by other problems.
         """
         rsync_255_fail = False
+        try:
+            platform_rsync_cmd = (
+                platform['rsync command']
+            )
+        except (TypeError, KeyError):
+            platform_rsync_cmd = 'rsync'
```

Comment:
* The KeyError should be handled by the parsec default
* The TypeError should be handled by the parsec validator.